hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/OMConfigKeys.java

```diff
@@ -85,7 +85,7 @@ private OMConfigKeys() {
 
   public static final String OZONE_KEY_DELETING_LIMIT_PER_TASK =
       "ozone.key.deleting.limit.per.task";
-  public static final int OZONE_KEY_DELETING_LIMIT_PER_TASK_DEFAULT = 1000;
+  public static final int OZONE_KEY_DELETING_LIMIT_PER_TASK_DEFAULT = 20000;
```

Comment:
Should also be changed in https://github.com/apache/ozone/blob/19dd94d594cfab0308b778dba367ecc62ed0f47e/hadoop-hdds/common/src/main/resources/ozone-default.xml#L396-L397

Reply:
Fixed.